sdk/eventhub/event-hubs/test/utils/testUtils.ts

```diff
@@ -0,0 +1,43 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+
+export const runnableInBrowserMarker = "#RunnableInBrowser";
+
+export const isNode = !!process && !!process.version && !!process.versions && !!process.versions.node;
+
+export enum EnvVarKeys {
+  EVENTHUB_CONNECTION_STRING = "EVENTHUB_CONNECTION_STRING",
+  EVENTHUB_NAME = "EVENTHUB_NAME",
+  IOTHUB_CONNECTION_STRING = "IOTHUB_CONNECTION_STRING"
+}
+
+const mandatoryEnvVars = [EnvVarKeys.EVENTHUB_CONNECTION_STRING];
+
+function throwMissingEnvironmentVariablesError(envVars: EnvVarKeys[]): void {
+  envVars.forEach(function(key: string) {
+    const name = key.valueOf();
+    if (!getEnvVarValue(name)) {
+      throw new Error(`Define ${name} in your environment before running integration tests.`);
+    }
+  });
+}
+
+function getEnvVarValue(name: string): string | undefined {
+  if (isNode) {
+    return process.env[name];
+  } else {
+    // @ts-ignore
+    return window.__env__[name];
+  }
+}
+
+export function getEnvVars(): { [key in EnvVarKeys]: any } {
+  // Throw error only if mandatory env variable is missing
+  throwMissingEnvironmentVariablesError(mandatoryEnvVars);
```

Comment:
The individual test files seem to be throwing error for when this even variable is missing along with check for event hub name. Therefore, we can skip the mandatoryEnvVars check here

Reply:
Should we instead remove the individual test file checks and have this written just once and checked at load time? This would be similar to what we have in Service Bus.

Reply:
All tests in Service Bus used the same connection string env variable. In Event Hubs we have some tests using connection string for Event Hubs, the other for IotHubs. So, the `mandatoryEnvVars` would be different based on whether we are testing using the Event Hubs connection string or the IotHub one.

To keep things simple, I would prefer to have these checks in the individual files i.e the way the are in the master branch

Reply:
I still think its simpler and makes sense to have it in the testUtils as opposed to individual files. The type of connection string value gets referenced by the Envirinment Variable key anyway. Not sure why the mandatory variables would change as they seem fixed?

We can agree to disagree though. :) Will skip the check in testUtils to unblock PR

Reply:
> Not sure why the mandatory variables would change as they seem fixed?

When one wants to run tests only from the iotHub.spec.ts file, the mandatory check in this PR in testUtils file will throw an error if the EventHub connection string is missing. Tests in the iotHub.spec.ts file need just the IotHub connection string and not the Event Hub connection string

Reply:
That makes sense! I had intended to manage mandatory envs centrally and not necessarily keep that check in.
If the intent is for developers to have flexibility in running tests without having to define all (unrelated) variables, we could also look into doing a lazy error handling at time of retrieving the value, as opposed to load time. 

Okay to keep changes simple for this PR (as maybe this can be revisited for Service Bus too, if needed)

Reply:
Updated and logged #3474